services/transaction_body.proto

```diff
@@ -363,22 +363,22 @@ message TransactionBody {
          * This transaction SHALL create a new consensus node record and add
          * that record to the network address book.
          */
-        com.hedera.hapi.node.addressbook.NodeCreateTransactionBody nodeCreate = 54;
+        NodeCreateTransactionBody nodeCreate = 54;
```

Comment:
Please do not break the correct packaging (both here and below).

Reply:
Updated.